scripts/install-worker.sh

```diff
@@ -305,23 +307,23 @@ fi
 sudo tar -xvf "${CNI_PLUGIN_FILENAME}.tgz" -C /opt/cni/bin
 rm "${CNI_PLUGIN_FILENAME}.tgz"
 
-sudo rm ./*.sha256
+sudo rm ./*.sha256 ./*.sha512
```

Comment:
I see... How about `sudo rm ./*.sha256 ./*.sha512` then? will it still be a conflict?